### PR TITLE
Readability of shortcuts #62

### PR DESCRIPTION
**Pull request by romanr**
I can't fix this myself;

image

Can't understand where it should be split so tokens like `Space` treated like single key.
In the end it should look like this: 

<kbd>⇧</kbd><kbd>⌘</kbd><kbd>SPACE</kbd>

### CODE REVIEW

**Comment by solomkinmv** [comment]
Thanks, I'll take a look!

**Comment by solomkinmv** [comment]
Sorry for the delay. I'll take a look at the end of the week!

**Comment by solomkinmv** [comment]
In this case the problem is because of the split.
"Space" gets converted to list of chars ["S", "p", "a", "c", "e"]. While you need to add spacing between modifiers and base key

**Comment by romanr** [comment]
> In this case the problem is because of the split. "Space" gets converted to list of chars ["S", "p", "a", "c", "e"]. While you need to add spacing between modifiers and base key

There's list of all keys (in key-codes.json?)
We need to iterate, in this example we have "⇧⌘SPACE", 
and iterating we will find first: shift `⇧` , and output it as singe <div>,
 then we will find `⌘` and output it, 
then "Space" is detected as single key because it is present in key-codes, so we output `SPACE` as one DIV representing one key.

File: shortcuts-disco-site/README.md [hunk 1]
```diff
@@ -5,6 +5,12 @@
 npm run dev
 ```
 
+If you're having error `next: Comamnd not found`:
+
+```bash
+npm install -g next
```
Comment (solomkinmv) [inline shortcuts-disco-site/README.md:11]: Are you sure that it needs to be global?

Based on the official docs you don't need it: https://nextjs.org/docs/getting-started/installation#manual-installation

**Comment by solomkinmv** [comment]
@romanr please see my last commit attached. I wrapped each keyboard token in span and added check if it's a symbol or not

File: shortcuts-disco-site/README.md (hunk 1 above)
Comment (romanr) [reply shortcuts-disco-site/README.md:11]: Yes `-g` is mistake, sorry.

**Comment by romanr** [comment]
> @romanr please see my last commit attached. I wrapped each keyboard token in span and added check if it's a symbol or not

Thanks, will check!

**Comment by solomkinmv** [comment]
I'll close this issue as stale. Please reopen it if you want to continue working on it.

**Pull request closed without merging**